// gitbook-plugin-toggle-chapters/index.js
module.exports = {
  website: {
    assets: "./book",
    js: [
      "toggle.js"
    ],
    css: [
      "toggle.css"
    ],
    html: {
      "html:start": function() {
        return "<!-- Start book "+this.options.title+" -->"
      },
      "html:end": function() {
        return "<!-- End of book "+this.options.title+" -->"
      },

      "head:start": "<!-- head:start -->",
      "head:end": "<script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create', 'UA-37765741-8', 'auto');ga('send', 'pageview');</script>",

      "body:start": "<!-- body:start -->",
      "body:end": "<!-- body:end -->"
    }
  },
  hooks: {
    // For all the hooks, this represent the current generator

    // This is called before the book is generated
    "init": function() {
      //console.log("init!");
    },

    // This is called after the book generation
    "finish": function() {
      //console.log("finish!");
    }

  }
};
